src/bundle/Resources/views/content/modal_location_bulk_operation_failed.html.twig

```diff
@@ -0,0 +1,46 @@
+<div
+    class="modal fade ez-bulk-operation-failed-modal"
+    id="bulk-operation-failed-modal"
+    tabindex="-1"
+    role="dialog">
+    <div
+        class="modal-dialog"
```

Comment:
This is less readable. If the line is really long it's fine to split it in few but in this case, (and in some other here) I think it's not needed.